core/templates/dev/head/pages/exploration-editor-page/editor-tab/exploration-editor-tab.directive.ts

```diff
@@ -135,6 +145,14 @@ angular.module('oppia').directive('explorationEditorTab', [
             }
           };
 
+          $scope.$on('stateEditorDirectiveInitialized', function(evt) {
+            if (ExplorationStatesService.isInitialized()) {
+              var stateData = (
+                ExplorationStatesService.getState(ctrl.stateName));
+              $rootScope.$broadcast('stateEditorInitialized', stateData);
+            }
+          });
```

Comment:
Should these be added to [this](https://github.com/oppia/oppia/blob/d25ff37869ebda62474509457f13129f7e51b451/core/templates/dev/head/components/question-directives/question-editor/question-editor.directive.ts#L111) function as well, in question editor? Since it deals with initialization and broadcasting the same data for questions.

Reply:
Looks like we already have a listener on 'stateEditorDirectiveInitialized' below which will call _init (which will broadcast stateEditorInitialized):
https://github.com/oppia/oppia/blob/d25ff37869ebda62474509457f13129f7e51b451/core/templates/dev/head/components/question-directives/question-editor/question-editor.directive.ts#L214-L216

So I think no changes are required.